resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -584,6 +595,128 @@ class TextCursor
     }
 }
 
+[DSCResource()]
+class ColorFilter
+{
+    # Key required. Do not set.
+    [DscProperty(Key)] [string] $SID
+    [DscProperty()] [nullable[bool]] $FilterStatus
+    [DscProperty()] [ColorFilters] $FilterColor
+    [DscProperty()] [nullable[bool]] $KeyboardShortcutStatus
+
+    static hidden [string] $FilterStatusProperty = 'Configuration'

```

Comment:
from looking at the registry, this doesn't seem needed.

Reply:
https://github.com/microsoft/winget-dsc/pull/66#discussion_r1802059124